README.md

```diff
@@ -52,6 +52,20 @@ abstract class MyPO {
 }
 ```
 
+**There are an addition two requirements for Dart files containing page objects:**
+1. Must be within either `/test/...` or `/testing/...` folder
+2. Must be suffixed with `_po.dart`
+
+For example: 
+1. `lib/po_containing_file.dart` will not work
```

Comment:
Put valid examples first.

For invalid examples, put a nested bullet explaining why.